src/pykeen/evaluation/evaluator.py

```diff
@@ -309,7 +309,7 @@ def evaluate(
         device = device or model.device
         tqdm_kwargs = dict(tqdm_kwargs or {})
         if not use_tqdm:
-            tqdm_kwargs.update(dict(disable=False))
+            tqdm_kwargs.update(dict(disable=True))
```

Comment:
The change looks good to me, but maybe it would be a good opportunity to also simplify this expression to

```suggestion
            tqdm_kwargs["disable"] = True
```

Reply:
Looks good!